src/checkers/inference/model/ArithmeticConstraint.java

```diff
@@ -57,6 +57,16 @@ public static ArithmeticOperationKind fromTreeKind(Kind kind) {
                     return OR;
                 case XOR:
                     return XOR;
+                case PLUS_ASSIGNMENT:
+                    return PLUS;
+                case MINUS_ASSIGNMENT:
+                    return MINUS;
+                case MULTIPLY_ASSIGNMENT:
+                    return MULTIPLY;
+                case DIVIDE_ASSIGNMENT:
+                    return DIVIDE;
+                case REMAINDER_ASSIGNMENT:
```

Comment:
There are a few more tree kinds where this handles the non-`_ASSIGNMENT` version, but not the `_ASSIGNMENT` version, e.g. `LEFT_SHIFT_ASSIGNMENT`. Should these be handled?
See https://github.com/openjdk/jdk/blob/master/src/jdk.compiler/share/classes/com/sun/source/tree/Tree.java for all kinds